specs/market-api.yaml

```diff
@@ -973,6 +999,9 @@ components:
         requestorId:
           type: string
           readOnly: true
+      required:
+        - demandId
+        - requestorId
```

Comment:
to make it required we need to define `NewDemand` and use it within `subscribeDemand` because it is the server to define those ids, not the client.

Reply:
That's fine, DemandOfferBase can be used as schema for requestBodies/Demand and requestBodies/Offer.